cmd/getroughtime/main.go

```diff
@@ -39,6 +39,7 @@ func main() {
 	configFile := flag.String("config", "", "A list of Roughtime servers.")
 	pingAddr := flag.String("ping", "", "Send a UDP request, e.g., localhost:2002.")
 	pingPubKey := flag.String("pubkey", "", "The Ed25519 public key of the address to ping.")
+	pingVersion := flag.String("ping-version", "", "The Roughtime version to use in ping.")
```

Comment:
I'd prefer a more explicit default:
```suggestion
	pingVersion := flag.String("ping-version", "Google-Roughtime", "The Roughtime version to use in ping.")
```